### PR TITLE
kindergarten-garden: Completionist

### PR DESCRIPTION
**Pull request by SleeplessByte**

### CODE REVIEW

File: exercises/kindergarten-garden/canonical-data.json [hunk 1]
```diff
@@ -66,43 +91,183 @@
       "cases": [
         {
           "uuid": "149b4290-58e1-40f2-8ae4-8b87c46e765b",
-          "description": "first student's garden",
+          "description": "for Alice, first student's garden",
```
Comment (ErikSchierboom) [inline exercises/kindergarten-garden/canonical-data.json:94]: I think we said that changing the description was a breaking change. I'm not totally sure though.

**Review by ErikSchierboom — COMMENTED** [review]
I like the completionist additions!

File: exercises/kindergarten-garden/canonical-data.json (hunk 1 above)
Comment (iHiD) [reply exercises/kindergarten-garden/canonical-data.json:94]: I think description is mutable as long as it clarifies or fixes typos. Which this does.

Comment (ErikSchierboom) [reply exercises/kindergarten-garden/canonical-data.json:94]: Agreed. I _think_ we forgot to document this :D

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/kindergarten-garden/canonical-data.json (hunk 1 above)
Comment (SleeplessByte) [reply exercises/kindergarten-garden/canonical-data.json:94]: We forgot to document this, but I remembered 🥇

**Review by valentin-p — APPROVED** [review]

**Review by petertseng — APPROVED** [review]
all new test cases added are correct. formatting not in its own commit is regrettable, but track maintainers can take comfort in the fact that immutable tests means that it's guaranteed that the test content has not changed, only the formatting

**Pull request merged**